### PR TITLE
Added test for Loader component

### LINKED ISSUE #91: (SP: 1) Write unit test for "Loader" component
### Component unit test
Unit test for "**Loader**" component
Scenaries descriptions:
- [ ] Component should be in the document

[Link to component](https://github.com/Space2Study-UA-1125/frontEnd/blob/develop/src/components/loader/Loader.jsx)

Current coverage:
Image

### PR DESCRIPTION
**Pull request by StasRusnak1**
Description: Component should be in the document
image

### CODE REVIEW

**Review by holus-bolus — APPROVED** [review]

**Review by VictoriaRodik — APPROVED** [review]

File: tests/unit/components/loader/Loader.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { render, screen } from '@testing-library/react'
+import Loader from '~/components/loader/Loader'
+
+describe('Loader test', () => {
+  test('should be in the document', () => {
```
Comment (iva-stasia) [inline tests/unit/components/loader/Loader.spec.jsx]: I was interested to find out [when it is better to use `it` and when to use `test`](https://daily-dev-tips.com/posts/jest-test-vs-it/). Maybe you'll find it useful too.

**Review by iva-stasia — APPROVED** [review]

File: tests/unit/components/loader/Loader.spec.jsx (hunk 1 above)
Comment (StasRusnak1) [reply tests/unit/components/loader/Loader.spec.jsx]: Thanks, I'll keep that in mind in the future.

**Review by OliaHrytsak — APPROVED** [review]

**Comment by OliaHrytsak** [comment]
We can see this Loader only when somethink 'isLoading'
Maybe you should test it conditionaly?

**Comment by StasRusnak1** [comment]
> We can see this Loader only when somethink 'isLoading' Maybe you should test it conditionaly?

Such condition is used in other components separately from Loader, so I think it's not necessary here

**Review by Wiret02 — APPROVED** [review]

File: tests/unit/components/loader/Loader.spec.jsx (hunk 1 above)
Comment (vlad-khrychov) [reply tests/unit/components/loader/Loader.spec.jsx]: I can add to this conversation. In one project we must use one approach and code style. So 
```suggestion
  it ('should be in the document', () => {
```

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]

**Review by ashfox21 — APPROVED** [review]

**Review by vlad-khrychov — APPROVED** [review]

**Pull request merged**